### ISSUE TITLE
Markdown and Html sidebar form fields won't focus / can't edit in 0.33.0

### ISSUE THREAD

**Issue by yannvanhalewyn** [tinacms/tinacms#1632]
## Description

Any rich form fields (Html / Markdown) in the sidebar can't be focused in order to edit.

It seems to be consistent on my machine with a minimal project. Bisecting all the versions degrading "tinacms" to "0.32.0" fixes the issue. Also, not sure if relevant, but removing the last class on the contentEditable div seemed to allow mouse events.

I'm setting up TinaCMS for the first time and this one took me a while to figure out since I might be making mistakes. Switching versions seems to be consistent in fixing the issue so it appears like a bug in tinacms 33. However this seems to be a **huge** problem so I'm also not sure why this issue hasn't been noticed by the devs / reported by users yet.

### Steps to reproduce

Basically an empty project with a single markdown field. My steps:

```sh
npx create-react-app tinabug
cd tinabug
yarn add tinacms react-tinacms-editor react-tinacms-inline styled-components
yarn start
```

```javascript
// src/index.js
import React from 'react'
import ReactDOM from 'react-dom'
import { TinaProvider, TinaCMS, useForm, usePlugin } from 'tinacms'
import { MarkdownFieldPlugin } from 'react-tinacms-editor'

const App = (props) => {
  const formConfig = {
    fields: [
      { name: "title", label: "Titel", component: "markdown" },
    ],
  };

  const [data, form] = useForm(formConfig);
  usePlugin(form);
  return <div>{data.title}</div>
}

const Root = ( { Component, pageProps }) => {
    const cms = new TinaCMS({
      enabled: true,
      sidebar: true,
      plugins: [MarkdownFieldPlugin]
    })

    return (
      <TinaProvider cms={cms}>
        <App {...pageProps} />
      </TinaProvider>
    )
}

ReactDOM.render(
  <React.StrictMode>
    <Root />
  </React.StrictMode>,
  document.getElementById('root')
);
```

### Expected result

When opening the sidebar I should be able to edit the title field.

### Actual result

Clicking on the title content yields no response.

#### Tinacms Package Versions

```
"react": "^17.0.1",
"react-dom": "^17.0.1",
"react-tinacms-editor": "^0.33.0",
"react-tinacms-inline": "^0.33.0",
"styled-components": "^5.2.1",
"tinacms": "^0.33.0",
```

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser | Chrome Latest        |
| os      |  MacOS Mojave 10.14.6  |
| node    | v14.4.0 |
| yarn     |    1.22.4     |

**Comment by kendallstrautman** [comment]
Thank you! I've confirmed this is only reproducible on Safari & Chrome. Investigating now....

**Comment by dwalkr** [comment]
This is fixed in 0.34.0. Thanks for the report!

@all-contributors add @yannvanhalewyn for bugs

**Comment by yannvanhalewyn** [comment]
Nice 👍 ! Thanks for the fix and the add 😉
